classes/helper.php

```diff
@@ -341,13 +341,16 @@ public static function retrieve_user_notices(): array {
                 continue;
             }
             $notice = $notices[$noticeid];
+            $dissmised = $data['action'] == acknowledgement::ACTION_DISMISSED;
```

Comment:
Is this just to make the code easier to understand?

Reply:
yes + coding styles